bundles/io/org.openhab.io.transport.cul/src/main/java/org/openhab/io/transport/cul/internal/serial/CULSerialHandlerImpl.java

```diff
@@ -130,6 +142,33 @@ protected void closeHardware() {
                 serialPort.close();
             }
         }
+    }
+
+    private void tryReopenHardware() {
+        closeHardware();
+        try {
+            openHardware();
+        } catch (CULDeviceException e) {
+            log.error("Failed to reopen serial connection after connection error", e);
+        }
+    }
+
+    @Override
+    protected void write(String command) {
+
+        try {
+            synchronized (bw) {
+                if (bw == null) {
```

Comment:
@9037568 fine with me. Should this and your follow-on PR target 1.9.1 or is that too risky?